source/reference/files/ondemand-d-ymls.rst

```diff
@@ -3,64 +3,165 @@
 ondemand.d/\*.yml files
 =======================
 
-Some configurations are held within yml files in the ``/etc/ood/config/ondemand.d/`` directory.
+Most of the configurations are now held within yml files in the ``/etc/ood/config/ondemand.d/`` directory.
 Open OnDemand will read all the ``.yml`` and ``.yml.erb`` files within this directory for
 configurations.
 
+Profiles
+-------------
+
+For more complex requirements, these configurations can be organized using profiles. With profiles,
+administrators can create different configuration settings within the same OnDemand installation.
+The selection of a profile can be configured to be manually selected by the user or automatically
+selected based on request data like the domain.
```

Comment:
We don't have a whole lot of verbiage here on the features. They mostly link back to the `customization.rst` page.

Can we follow the pattern for `pinned_apps` where most of the docs are on `customization` and we simply link back to there for more information.

Reply:
OK - will update

Reply:
Updated. Added a new placeholder section to the customization page to explain profiles. Will complete this section in another PR.

I have added all `boolean` and `string` properties from ConfigurationSingleton. I completed the ones I know about. Left the others as placeholders. Not sure I should keep them.